Lib/dataclasses.py

```diff
@@ -1122,6 +1123,20 @@ def _dataclass_setstate(self, state):
         object.__setattr__(self, field.name, value)
 
 
+def _get_slots(cls):
+    match cls.__dict__.get('__slots__'):
```

Comment:
Look at you getting all fancy with a `match` statement! Nice.

Reply:
It feels really good not to have to write 3.9< compatible code haha 😄